### DISCUSSION TITLE
Why not merging features to ViMusic ?

### DISCUSSION THREAD

**Discussion by KaKi87** [fast4x/RiMusic#375] [General]
Hi,

I'm wondering what's the reason why this fork and its interesting features are distributed in a separate product instead of merging these in ViMusic ?

Also, it would be nice to highlight those features that are differenciating Ri from Vi in the README.

Thanks !

**Comment by ikanakova** [comment]
As far as I know, ViMusic is abandoned by its devoper, so it would be pointless to do pull requests because the merge would never happen.

**Reply by ikanakova** [reply]
Distinguishing the new RiMusic features from the original ViMusic features in the Readme is a good idea.

**Reply by KaKi87** [reply]
Indeed, well I guess I missed it because I'm not actually using either app, but recommended Vi from time to time to people who'd be interested in it, until someone made me discover Ri today.

I suppose I'll be recommending Ri from now on. ^^

**Reply by KaKi87** [reply]
> Distinguishing the new RiMusic features from the original ViMusic features in the Readme is a good idea.

Well, now knowing that Vi is unmaintained, I don't find differenciating from it to be as much useful anymore, since this difference alone makes it an obvious choice. ^^

**Comment by ghost** [comment]
This'd make a good entry in the FAQs. As soon as I see "forked from ..." on a repo, I wonder why.

**Reply by ikanakova** [reply]
> This'd make a good entry in the FAQs. As soon as I see "forked from ..." on a repo, I wonder why.

I don't think so. The readme clearly states "It was born as an extension of the functionality of ViMusic, which retains the general appearance, colors." Forks are, among other things, to take the original application and then independently develop further.

**Reply by KaKi87** [reply]
Usually not independently, no, but rather to submit PRs, hence my question.

Which have been answered of course, but having the answer in README instead of having to look for it here would be better indeed.

Most projects don't need an FAQ entry though : "forked from the abandoned ViMusic" should suffice.

**Reply by ikanakova** [reply]
> Usually not independently, no, but rather to submit PRs, hence my question.

No, that's definitely not true - see https://en.wikipedia.org/wiki/Fork_(software_development)

**Reply by KaKi87** [reply]
From the very page you linked :

> GitHub uses "fork" as its term for this method of contribution to a project.

**Reply by ikanakova** [reply]
According to [GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo), forks serve two purposes: _Propose changes to someone else's project_ & _**Use someone else's project as a starting point for your own idea**_. That's the end of our discussion, because this is pointless, I think most people on GitHub understand what a fork is, so there's no point in explaining it in the readme or faq.

**Reply by KaKi87** [reply]
There's a reason why the part you highlighted is the second one : the first one is more frequent.

So, when the second happens, it does for a reason, which is good to be explained in READMEs.

Feel free not to reply, BWPanda and I already made our point.
